### PR TITLE
Sprint12 staging

### PR DESCRIPTION
**Pull request by jeremythuff**
#197 - The last nav-list-item of tamu-header component is not visible on Firefox (mobile view).
#200 - TL Components should provide a branded tl-card component.
#201 - TL Components should provide a branded tl-panel component.
#202 - TL Components should expose wvr-usage docs.
#204 - TL Components should be refactored to use the wvre slector and the wvr-*-component selector.
#217 - Lato font not being read into mega menu component.
#220 - The tl-header should use the configured asset path for the header logo.
#223 - Remove drop shadow from mega menu section titles.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,11 +51,18 @@
     "@angular/platform-browser-dynamic": "^10.0.12",
```
Comment (wwelling) [inline package.json:51]: Probably should keep up to date angular. At least match version in weaver-components.

File: scripts/build-docker.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#! /usr/bin/env node
```
Comment (wwelling) [inline scripts/build-docker.js:1]: I do not think there should be a space between `#!` the path.

File: scripts/build-tl-components-static.js [hunk 3]
```diff
@@ -1,3 +1,5 @@
+#! /usr/bin/env node
```
Comment (wwelling) [inline scripts/build-tl-components-static.js:1]: Will these node scripts continue to work on windows with a Linux shebang?

**Review by wwelling — APPROVED** [review]
Everything looks good from what I can tell. Minor recommendation for matching weaver components version of angular.

File: package.json (hunk 1 above)
Comment (jeremythuff) [reply package.json:51]: We will upgrade in the next sprint.

**Pull request merged**